lib/bolt/core/src/dep/nomad/gen.rs

```diff
@@ -519,6 +523,7 @@ pub async fn gen_svc(region_id: &str, exec_ctx: &ExecServiceContext) -> Job {
 				log_config: Some(json!({
 					"MaxFiles": 2,
 					"MaxFileSizeMB": 4,
+					"Disabled": project_ctx.ns().logging.is_none(),
```

Comment:
I think this is backwards, should be `"Disabled": project_ctx.ns().logging.is_some()` since we don't want the default Nomad logging config if we're using the Loki log shipping.

Can you double check this?

Reply:
Looks like `Disabled` was added in Nomad 1.5. We're on 1.4, so this option has no effect.

Reply:
1c927a34860586a6cae6300662caa698476961e6